integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+# OpenID Connect Example with Server Side Web App
+## Overview
+This example application uses the OpenID Connect (OIDC) protocol to authenticate
+with Azure Active Directory and get an access token to request WorkflowGen's
+GraphQL API.
```

Comment:
This sample application uses the OpenID Connect (OIDC) protocol to authenticate with Azure Active Directory and get an access token to request WorkflowGen's GraphQL API.